api/tests/test_add_response_headers.py

```diff
@@ -0,0 +1,25 @@
+from api.security.headers import SECURITY_HEADERS, CORS_HEADERS
```

Comment:
issue: As tedious as it might seem, we shouldn't test against the actual configuration object used by the software under test

This is usually avoided because you'd be replicating the same logic that you are testing, nullifying the test.

Practical example:
- I go in and remove an important header from the config under test
- The test is green and nobody notices

One solution would be enumerating the headers in the test as well

Reply:
Absolutely right, done in next revision! Thank you!